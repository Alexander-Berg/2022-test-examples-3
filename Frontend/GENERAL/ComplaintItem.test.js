import React from 'react';
import { render } from 'enzyme';

import ComplaintItem from './ComplaintItem';

describe('Should render', () => {
    it('default ComplaintModal', () => {
        const wrapper = render(
            <ComplaintItem
                className={'Block-Element'}
                complaint={{
                    message: 'Message',
                    createdAt: new Date('2019-09-17T00:15:16Z'),
                }}
            />
        );

        expect(wrapper).toMatchSnapshot();
    });
});
