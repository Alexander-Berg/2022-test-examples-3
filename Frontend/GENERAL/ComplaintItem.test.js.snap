// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render default ComplaintModal 1`] = `
<div
  class="ComplaintItem Block-Element"
>
  <div
    class="ComplaintItem-Message"
  >
    Message
  </div>
  <div
    class="ComplaintItem-Data"
  >
    17 сент. 2019
  </div>
</div>
`;
